Makefile

```diff
@@ -189,7 +189,6 @@ clobber: clean
 	-rm -rf watt
 	-$(if $(filter-out -,$(ENVOY_COMMIT)),rm -rf envoy envoy-src)
 	-rm -rf docs/node_modules
-	-rm -rf .skip_test_warning	# reset the test warning too
```

Comment:
Hmmm, I almost missed this. Can you take this out of this PR, please? I'm not necessarily averse to this change as a separate thing, but let's not bury a change to this build semantic in this. 🙂